kentikapi/users_integration_test.go

```diff
@@ -195,114 +195,17 @@ func TestClient_GetAllUsers(t *testing.T) {
 	}
 }
 
-func TestClient_GetUser_WithTimeout(t *testing.T) {
+func TestClient_GetUser(t *testing.T) {
 	tests := []struct {
-		name            string
-		responses       []testutil.HTTPResponse
+		name                string
+		responses           []testutil.HTTPResponse
+		serverHandlingDelay time.Duration
+		// timeout and expected requests are strictly connected. The server sleeps for 10 ms before each response.
```

Comment:
```suggestion
```
Please just remove remove that comment. The second sentence is false for multiple test cases. And the first one doesn't explain anything (at least for me)

Reply:
Done